app/forms/SignUpForm.scala

```diff
@@ -75,6 +92,26 @@ object SignUpForm {
     override def unbind(key: String, value: String): Map[String, String] = Map(key -> value)
   }
 
+  val applicationRouteFormatter = new Formatter[String] {
+    override def bind(key: String, data: Map[String, String]): Either[Seq[FormError], String] = {
+      val appRoute = data.getOrElse(key, "")
```

Comment:
why do we have default=""? It will fail the enum transformation withName("") => NoSuchElement